corehq/apps/locations/resources/v0_5.py

```diff
@@ -59,6 +59,10 @@ def get_resource_uri(self, bundle_or_obj=None, url_name='api_dispatch_list'):
             'pk': obj.pk
         })
 
+    def dehydrate_parent(self, bundle):
+        if parent := bundle.obj.parent_type:
```

Comment:
That expression caught me for a bit.
I don't how that is better than
```
if bundle.obj.parent_type:
    return self.get_resource_uri(bundle.obj.parent_type)
```

Plus,
I wonder if this and even the other use of this expression, is best suited here, probably because the target value is already available by another means.

Reply:
Yeah, the `:=` is a relatively new python operator, but I've started getting a feel for it - nice to make things a little bit shorter.  Here these `dehydrate` methods all concern themselves with a specific field value (eg `parent`, `location_type`), so it made sense to me to pull that value into a local variable, but I can see how it looks a bit weird, so I'll change it.

Reply:
Here `obj` is a location type, and `parent_type` is a related field - a `LocationType` instance.  If you expand the diff you can see how it's defined on the resource.

Reply:
> but I can see how it looks a bit weird, so I'll change it.

Thank you :+1: 

Cool, I think the naming "parent == bundle.obj.parent_type" that got me.
I wish it was named parent_location_type instead.